backend/src/Core/FurnitureShop.Core.Contracts/Web/Blobs/GetAllPhotosUrls.cs

```diff
@@ -2,7 +2,7 @@
 using LeanCode.CQRS;
 using LeanCode.CQRS.Security;
 
-namespace FurnitureShop.Core.Contracts.Mobile.Blobs
+namespace FurnitureShop.Core.Contracts.Web.Blobs
 {
     [AuthorizeWhenHasAnyOf(Auth.Roles.Admin)]
     public class GetAllPhotosUrls : IRemoteQuery<List<string>>
```

Comment:
I don't think it will be use anywhere, maybe I missed sense of this Query

Reply:
nie wiem czy będzie potrzebne ale na przykład: admin chce dodać nowy produkt, nie chce dodawac nowego zdjęcia tylko wybrać któreś z istniejących, no to potrzebuje pobrania dostępnych